sbe-tool/src/main/java/uk/co/real_logic/sbe/ir/Token.java

```diff
@@ -105,6 +107,7 @@
     public Token(
         final Signal signal,
         final String name,
+        final String packageName,
```

Comment:
Should come at end of string args as new.

Reply:
Done.